daemon/endpoint.go

```diff
@@ -62,34 +62,34 @@ func (h *getEndpoint) Handle(params GetEndpointParams) middleware.Responder {
 		endpointmanager.Mutex.RUnlock()
 		wg.Wait()
 		return NewGetEndpointOK().WithPayload(eps)
-	} else {
-		eps := []*models.Endpoint{}
+	}
 
-		// Convert params.Labels to model that we can compare with the endpoint's labels.
-		convertedLabels := labels.NewLabelsFromModel(params.Labels)
+	eps := []*models.Endpoint{}
 
-		endpointmanager.Mutex.RLock()
+	// Convert params.Labels to model that we can compare with the endpoint's labels.
+	convertedLabels := labels.NewLabelsFromModel(params.Labels)
 
-		wg.Add(len(endpointmanager.Endpoints))
-		for k := range endpointmanager.Endpoints {
-			go func(wg *sync.WaitGroup, ep *endpoint.Endpoint) {
-				ep.Mutex.RLock()
-				if ep.HasLabels(convertedLabels) {
-					eps = append(eps, ep.GetModel())
-				}
-				ep.Mutex.RUnlock()
-				wg.Done()
-			}(&wg, endpointmanager.Endpoints[k])
-		}
-		endpointmanager.Mutex.RUnlock()
-		wg.Wait()
+	endpointmanager.Mutex.RLock()
 
-		if len(eps) == 0 {
-			return NewGetEndpointNotFound()
-		}
+	wg.Add(len(endpointmanager.Endpoints))
+	for k := range endpointmanager.Endpoints {
+		go func(wg *sync.WaitGroup, ep *endpoint.Endpoint) {
+			ep.Mutex.RLock()
+			if ep.HasLabels(convertedLabels) {
+				eps = append(eps, ep.GetModel())
+			}
+			ep.Mutex.RUnlock()
+			wg.Done()
+			}(&wg, endpointmanager.Endpoints[k])
```

Comment:
Can you run `gofmt -w daemon/endpoint.go`? The verifier is failing because of this line.

Reply:
Rebased with exec to run gofmt.